app/src/main/java/com/example/pygmyhippo/organizer/EventFragment.java

```diff
@@ -295,16 +295,14 @@ private void populateAllFields() {
         eventAboutDescriptionView.setText(event.getDescription());
 
         // If the event is closed, then change the style of the draw button (and there are no available spots)
-        if (!event.hasAvailability()) {
-            if (event.getEventStatus().value.equals("cancelled")) {
+        if (event.getEventStatus().value.equals("cancelled")) {
```

Comment:
NIT: instead of using the 'cancelled` string, you could do something like `Event.EventStatus.cancelled`